telemetrylib/DTI.h

```diff
@@ -0,0 +1,43 @@
+//
+// Created by Mingcan Li on 1/22/23.
+//
+
+#ifndef TELEMETRYLIB_DTI_H
+#define TELEMETRYLIB_DTI_H
+
+#include <QtCore>
+#include <QtNetwork>
+
+class DTI : public QObject{
+    Q_OBJECT
+public:
+    /**
+     * Receive bytes via channel to be implemented
+     * @return bytes received
+     */
+    virtual std::string receiveData() = 0;
```

Comment:
See comment in `library.cpp`, but I think this should be implemented as a method for requesting data rather than reading a response, as the readReply() slot is used to read responses

Reply:
See response. Could leave reimplementation as a future task and remove or at least comment out this function.